src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -1149,6 +1149,161 @@ def test_aks_update_to_msi_cluster_with_addons(self, resource_group, resource_gr
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
 
+    # # These tests require subscription-level feature flags to be set by AKS and Azure Monitor. The features will be 
+    # # broadly rolling out soon, then these tests can be uncommented. (written on 6/29/2021)
```

Comment:
Could you please uncomment the test and filter out this test case by adding its name to the filter defined [here](https://github.com/daweim0/azure-cli-extensions/blob/david/aks-monitoring-aad-auth/src/aks-preview/azcli_aks_live_test/ext_matrix_default.json#L29-L30) in your repo or [here](https://github.com/Azure/azure-cli-extensions/blob/main/src/aks-preview/azcli_aks_live_test/configs/ext_matrix_default.json#L29-L30) in the latest main branch?

Reply:
done, thanks for explaining how to fix the issue